### PR TITLE
AC-679 Added SwipeToRefresh Feature in all the necessary Activities

### PR DESCRIPTION
**Pull request by rishabh-997**
## Description of what I changed
I added swipe to refresh the layout feature in the following activities:
1. FindPatient
2. Find ActiveVisits
3. Find FormEntry
4. Find Providers

## Issue I worked on

JIRA Issue: https://issues.openmrs.org/browse/AC-679

## Checklist: I completed these to help reviewers :)

- [x] My pull request only contains **ONE single commit**
(the number above, next to the 'Commits' tab is 1).

- [x] I have **added tests** to cover my changes. (If you refactored
existing code that was well tested you do not have to add tests)

- [x] All new and existing **tests passed**.

- [x] My pull request is **based on the latest changes** of the master branch.

### CODE REVIEW

**Comment by rishabh-997** [comment]
@f4ww4z sir

File: openmrs-client/src/main/res/layout/fragment_active_visits.xml [hunk 1]
```diff
@@ -12,28 +12,35 @@
...
-    <androidx.recyclerview.widget.RecyclerView
-        android:id="@+id/visitsRecyclerView"
-        android:layout_width="match_parent"
-        android:layout_height="match_parent"
-        />
+        <androidx.recyclerview.widget.RecyclerView
+            android:id="@+id/visitsRecyclerView"
+            android:layout_width="match_parent"
+            android:layout_height="match_parent"
+            />
```
Comment (Abdelaty) [inline openmrs-client/src/main/res/layout/fragment_active_visits.xml]: Make sure to reformat the files after you finish edits

File: openmrs-client/src/main/res/layout/fragment_provider_management.xml [hunk 2]
```diff
@@ -23,10 +23,17 @@
         android:layout_centerInParent="true"
         android:visibility="visible" />
 
-    <androidx.recyclerview.widget.RecyclerView
-        android:id="@+id/providerManagementRecyclerView"
+    <androidx.swiperefreshlayout.widget.SwipeRefreshLayout
+        android:id="@+id/swipeLayout"
         android:layout_width="match_parent"
-        android:layout_height="match_parent" />
+        android:layout_height="match_parent">
```
Comment (Abdelaty) [inline openmrs-client/src/main/res/layout/fragment_provider_management.xml]: @rishabh-997 Make sure that your PR based on the latest changes of the master branch.

**Comment by Abdelaty** [comment]
I left some comments to you on commits. please check it.
And don't check tasks in checklist that you didn't did it. @rishabh-997

**Comment by rishabh-997** [comment]
@f4ww4z sir, reviews to this PR

File: openmrs-client/src/main/res/layout/fragment_synced_patients.xml [hunk 3]
```diff
@@ -25,11 +25,17 @@
         android:layout_width="wrap_content"
         android:layout_height="wrap_content" />
 
-    <androidx.recyclerview.widget.RecyclerView
-        android:id="@+id/syncedPatientRecyclerView"
+    <androidx.swiperefreshlayout.widget.SwipeRefreshLayout
+        android:id="@+id/swipeLayout"
         android:layout_width="match_parent"
-        android:layout_height="match_parent"
-        />
```
Comment (AryanGanotra07) [inline openmrs-client/src/main/res/layout/fragment_synced_patients.xml]: Please keep the xml formatted.

**Comment by VibhorChinda** [comment]
LGTM nice work @rishabh-997

**Pull request merged**